### ISSUE TITLE
Regional Conditioning ignores masks

### ISSUE THREAD

**Issue by EightiesPower** [logtd/ComfyUI-Fluxtapoz#54]
I've run the provided flux_regional workflow, with all the same models, and out of the 6 or so generations I did, the UFO wasn't dead center of the image exactly once.  When I changed the prompt to have two subjects instead of one, the model would tend to just mash them together, resulting in often hilarious hybrids.
I've updated Comfy and the nodes before trying this. Is there anything that can be done to improve mask conditioning without using controlnets/Redux? Because currently I'm getting better results with the old MultiAreaConditioning node, which wasn't even designed for Flux.

**Comment by logtd** [comment]
You can change the attention override to target more attention layers. The more that are targeted the more the prompt is confined to the masked area.

There are 0-18 double layers and 0-36 single layers

**Comment by EightiesPower** [comment]
I've done that (I think) and it seems to have worked, thanks! Just so I understand what I'm doing better - in the node the top textbox is for the double layers and the bottom one for single layers?

**Comment by logtd** [comment]
yep, that's correct. You write the block indexes out separated by commas, e.g.
```
0,1,2,3,4,5
```
```
30,31,32,33,34,35
```
just randomly chose those numbers, no significance

**Comment by drmbt** [comment]
even unplugging this and turning end up to 1, i can't get the mask to do anything. can you confirm this works, and maybe update that workflow with an obvious example?
